.pre-commit-config.yaml

```
@@ -24,14 +24,14 @@ repos:
       - id: pyproject-fmt
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.8.0
+    rev: v0.9.0
```

Ruff 0.9.1 came out with some fixes, maybe it's fine to add it :)

```suggestion
    rev: v0.9.1
```